src/server/src/main/java/io/cassandrareaper/resources/ClusterResource.java

```diff
@@ -84,21 +85,26 @@ public ClusterResource(AppContext context, ExecutorService executor) {
   }
 
   @GET
-  public Response getClusterList(@QueryParam("seedHost") Optional<String> seedHost)
-      throws ReaperException {
-
+  public Response getClusterList(@QueryParam("seedHost") Optional<String> seedHost) throws ReaperException {
     LOG.debug("get cluster list called");
-    Collection<Cluster> clusters = context.storage.getClusters();
+    Collection<Cluster> clusters = removeConflictingClusters(context.storage.getClusters());
```

Comment:
I'm worried by the additional cost we're adding on getting the cluster list, especially for large installs.
Having conflicting clusters is not something that should happen that much, unless you have a large infrastructure with IP addresses that get re-used a lot.
I'd be in favor of creating a new rest operation, or run the command in the background on a regular basis (once per day for example, or upon startup of Reaper), instead of doing this check every time we list the clusters. We list them A LOT in the UI.